app/main/autoupdater.ts

```diff
@@ -1,18 +1,19 @@
 'use strict';
-const { app, dialog, shell } = require('electron');
-const { autoUpdater } = require('electron-updater');
-const isDev = require('electron-is-dev');
+import { app, dialog, shell } from 'electron';
+import { autoUpdater } from 'electron-updater';
+import { linuxUpdateNotification } from './linuxupdater';	// Required only in case of linux
 
-const ConfigUtil = require('./../renderer/js/utils/config-util.js');
+import log = require('electron-log');
+import isDev = require('electron-is-dev');
+import ConfigUtil = require('../renderer/js/utils/config-util');
 
-function appUpdater(updateFromMenu = false) {
+export function appUpdater(updateFromMenu = false): void {
 	// Don't initiate auto-updates in development
 	if (isDev) {
 		return;
 	}
 
 	if (process.platform === 'linux' && !process.env.APPIMAGE) {
-		const { linuxUpdateNotification } = require('./linuxupdater');
```

Comment:
@priyank-p, @vsvipul I put it here because I thought we'll only require/import this on Linux.

Reply:
OK, this change was made because linting usually complains of assigning value through require statement in typescript unless it's a side effect require i.e. `require('some-module')` not `const someModule = require('some-module')`. Since there is nothing in the `linux-updater.js` that causes error if the os is not linux I think we should be fine with this as-is?

Reply:
Yeah, there are no side-effects. I was just worried about the performance since we are importing this on other systems also.